src/reynir_correct/checker.py

```diff
@@ -247,10 +247,15 @@ def annotate(self, sent: Sentence) -> List[Annotation]:
                     assert original_terminal is not None
                     terminal = grammar.terminals[original_terminal]
                     assert isinstance(terminal, VariantHandler)
-                    if t.suggestion_does_not_match(terminal, bin_token):
-                        # If this token is annotated with a spelling suggestion,
-                        # do not add it unless it works grammatically
+                    try:
+                        if t.suggestion_does_not_match(terminal, bin_token):
+                            # If this token is annotated with a spelling suggestion,
+                            # do not add it unless it works grammatically
+                            annotate = False
+                    except AttributeError:
+                        # 1,8 milljarður kóna - wrap_tokens seems to fail
                         annotate = False
+                        #print("Meaning type doesn't match BinToken: {}, type {}".format(bin_token, type(bin_token.t2[0])))
```

Comment:
Delete this

Reply:
Deleted. Wanted to keep it as a diagnostic comment but mypy didn't approve.